xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -963,8 +961,18 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
   {
     StringUtils::ToLower(strTmp);
     if (strTmp == "horizontal")
+    {
       orientation = HORIZONTAL;
+      // default orientation for a settings slider is vertical
+      // consequently it has to changed here
+      float tmpVal = sliderWidth;
```

Comment:
it is very unclear that you change the initial values set above. Took me quite a while to see this.